runtime/lib/ttnn/runtime.cpp

```diff
@@ -175,6 +177,8 @@ Event submit(Device deviceHandle, Binary executableHandle,
   return Event(nullptr);
 }
 
-void wait(Event) { throw std::runtime_error("Not implemented"); }
+void wait(Event) {
+  // Not implemented
```

Comment:
Yes because otherwise a FE runtime cannot generically use this interface.  It's perfectly legal for a backend runtime to implement `submit` as blocking and `wait` is a nop. Which is how ttnn runtime works today, but we should fix this and use real ttnn events and nonblocking APIs